### PR TITLE
Issue/137 fix r14 funding distribution

### LINKED ISSUE #137: Fix issues for R14 that caused a miscount in funding distribution
There is an issue in R14 that is causing a funding distribution problem.

### DoD:
- [ ] Setup a functional test that takes R14 data from Airtable, fixes the issue, and verifies that other functionality is still working.

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #137.

### CODE REVIEW

File: src/airtable/rounds/funding_rounds.js [hunk 1]
```diff
@@ -153,8 +158,8 @@ const calculateWinningProposalsForEarmark = (
       p.fields['USD Granted'] = oceanGranted * oceanPrice
       p.fields['OCEAN Granted'] = oceanGranted
       p.fields['Proposal State'] = 'Granted'
-      fundsLeft =
-        fundsLeft - usdGranted <= 0 ? fundsLeft : fundsLeft - usdGranted
+      fundsLeft = fundsLeft - usdGranted <= 0 ? 0 : fundsLeft - usdGranted
```
Comment (trizin) [inline src/airtable/rounds/funding_rounds.js:161]: Why not use `fundsLeft - usdGranted <= 0 ? fundsLeft`?

Comment (KatunaNorbert) [reply src/airtable/rounds/funding_rounds.js:161]: If we can't fully grant we will give all the remaining funds which will lead to `fundsLeft = 0` , before we had `fundsLeft = fundsLeft` which means that we give the funds but the funds will still remain inside funds left.

Comment (trizin) [reply src/airtable/rounds/funding_rounds.js:161]: That makes perfect sense!

**Review by trizin — APPROVED** [review]

File: src/scripts/update_funding_round.js [hunk 2]
```diff
@@ -298,7 +298,125 @@ const main = async () => {
...
+      gsheetRows.push(oceanResultsValues)
+      gsheetRows.push([])
+
+      await updateValues(
+        oAuth,
+        sheetName,
+        'A1:I' + (gsheetRows.length + 1),
+        gsheetRows
+      )
+    }
```
Comment (idiom-bytes) [inline src/scripts/update_funding_round.js:419]: Will look at this a bit closer.

**Pull request merged**